src/app/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { UserDetailService } from '../services/user-detail.service';
+import { Router } from '@angular/router';
+import { Title } from '@angular/platform-browser';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+import { EmployeeDataService } from '../services/employee-data.service';
+import { EmployeeDetailService } from '../services/employee-detail.service';
+
+
+// import { User } from './../../models/user';
+// import { AppState, selectAuthenticationState } from './../../store/app.state';
+// import { Logout } from './../../store/actions/authentication.actions';
+// import { UserService } from 'src/app/services/user.service';
+// import { FileUploadService } from 'src/app/services/file-upload.service';
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.scss']
+})
+
+export class DashboardComponent implements OnInit {
+  empName: string;
+  getState: Observable<any>;
+  isAuthenticated: boolean = false;
+  fetchDone = false;
+  users: string;
+  employeeList: string;
+  month = new Date().getMonth();
+  months: string[] = ["January", "February", "March", "April", "May",
+    "June", "July", "August", "September", "October", "November", "December"];
+  currentMonth = this.months[this.month];
+  years: number[] = [2017, 2018, 2019];
```

Comment:
`years` array should hold range from prev to current running year. If will fail if the year will 2020. Please, those cases also.